build.sbt

```diff
@@ -892,6 +892,7 @@ lazy val test = project
     IntegrationTest / fork := true,
     Compile / scalacOptions += "-Yvalidate-pos:parser,typer",
     IntegrationTest / javaOptions ++= List("-Xmx2G", "-Dpartest.exec.in.process=true", "-Dfile.encoding=UTF-8", "-Duser.language=en", "-Duser.country=US") ++ addOpensForTesting,
+    IntegrationTest / javaOptions ++= { if (scala.util.Properties.isJavaAtLeast("18"/*18*/)) List("-Djava.security.manager=allow") else Nil },
```

Comment:
what's the comment for?

Reply:
If we weren't stuck on 2.12, I could use an ordinary `Int`. Numbers are so beautiful. Commenting it registers my dissatisfaction or disaffection without requiring me to go out in the streets with a sandwich board.

Reply:
removing the comment as commentary.

Reply:
Moved that commit to a clean PR with the implied suggested changes.